README.md

```diff
@@ -49,8 +54,8 @@ This script translates input text into gloss notation.
 text_to_gloss \
   --text <input_text> \
   --glosser <simple|rules|nmt> \
-  --spoken-language <de|fr|it|en> \
-  --signed-language <sgg|gsg|bfi>
+  --spoken-language <de|fr|it> \
+  --signed-language <sgg|fsl|ise>
```

Comment:
In the paper, we claim swiss-french and swiss-italian (`ssr` and `slf`), and these indeed are the supported language codes.